spec/features/anonymous_user_happy_spec.rb

```diff
@@ -22,8 +34,28 @@
   end
 
   scenario 'they expected the most popular links' do
+    url_1 = Url.create!(original: 'https://www.twitter.com', popularity: 8)
+    url_2 = Url.create!(original: 'https://www.google.com', popularity: 4)
+    url_3 = Url.create!(original: 'https://www.facebook.com', popularity: 9)
+    expected_popular = [url_3.original, url_1.original, url_2.original]
+
+    visit root_path
+    actual_popular = all('#popular td').map(&:text)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.